modules/rostests/kmtests/kmtest_drv/kmtest_fsminifilter.c

```diff
@@ -543,12 +547,12 @@ FilterPostOperation(
 {
     FLT_POSTOP_CALLBACK_STATUS Status;
     UCHAR MajorFunction;
-    INT i;
+    ULONG i;
 
     Status = FLT_POSTOP_FINISHED_PROCESSING;
     MajorFunction = Data->Iopb->MajorFunction;
 
-    for (i = 0; i < sizeof(Callbacks) / sizeof(Callbacks[0]); i++)
+    for (i = 0; i < CallbacksCount; i++)
```

Comment:
{{
c:\reactos-cov\modules\rostests\kmtests\kmtest_drv\kmtest_fsminifilter.c(551,18) :  warning: comparison of unsigned expression < 0 is always false [-Wtautological-unsigned-zero-compare]
    for (i = 0; i < sizeof(Callbacks) / sizeof(Callbacks[0]); i++)
                ~ ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
}}